### ISSUE TITLE
Landmarks - Multiple "main"

### ISSUE THREAD

**Issue by katknow** [omeka-s-themes/cozy#38]
Currently, the theme is failing the Landmarks test on Accessibility Insights for Web as there are two sections labeled as "main" when viewing the homepage in Chrome (it's fine in Edge).

**Comment by kimisgold** [comment]
I've reduced the number of main landmarks to a single content area. Ready to test.

**Comment by katknow** [comment]
This now passes the AIW test for landmarks and is good to go
